From 5c3606dc5bcdee8ef2d13aab4fb06f0d60124096 Mon Sep 17 00:00:00 2001
From: Tinghui WANG (Steven) <steven.wang@digilentinc.com>
Date: Wed, 19 Dec 2012 15:42:06 -0800
Subject: [PATCH] pmods: Add GPL License Header

Add Copyright information and GPL license header to PmodOLED driver.

Signed-off-by: Tinghui WANG (Steven) <steven.wang@digilentinc.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/pmods/pmodoled-gpio.c |   23 ++++++++++++++++++++++-
 1 files changed, 22 insertions(+), 1 deletions(-)

diff --git a/drivers/pmods/pmodoled-gpio.c b/drivers/pmods/pmodoled-gpio.c
index 5cd330b..f6128e7 100644
--- a/drivers/pmods/pmodoled-gpio.c
+++ b/drivers/pmods/pmodoled-gpio.c
@@ -1,3 +1,24 @@
+/*
+ * pmodolde-gpio.c - PmodOLED-GPIO driver
+ *
+ * Copyright (c) 2012 Digilent.  All right reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ *
+ */
+
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/platform_device.h>
@@ -838,6 +859,6 @@ static struct platform_driver gpio_pmodoled_driver = {
 module_platform_driver(gpio_pmodoled_driver);
 
 MODULE_AUTHOR("Digilent, Inc.");
-MODULE_LICENSE("GPL");
+MODULE_LICENSE("GPL v2");
 MODULE_DESCRIPTION(DRIVER_NAME": PmodOLED display driver");
 MODULE_ALIAS(DRIVER_NAME);
-- 
1.7.1

